content/sensu-go/5.0/api/overview.md

```diff
@@ -43,6 +43,8 @@ The Sensu API requires authentication using a JWT access token.
 Sensuctl provides an easy way to generate access tokens for short-lived use with the Sensu API.
 The user credentials that you use to log in to senscutl determine your permissions to get, list, create, update, and delete resources using the Sensu API.
 
+_NOTE: The [`/health`][5] API endpoint does not require authorization._
```

Comment:
I remember referring to this as a "note", so apologies for that. However, I don't think this information really fits in with our definition for NOTE callouts. I'd recommend integrating this into the preceding paragraph. Perhaps something like: `With the exception of the [health API][5], the Sensu API requires...`


Also, just noticed `senscutl` on line 44 (_facepalm_). If you're able to fix that as well, I'd be forever grateful 🙏

Reply:
Added the typo fix.